### PR TITLE
Symfony 4 support

### PR DESCRIPTION
**Pull request by chirimoya**
https://github.com/phpcr/phpcr-migrations/pull/8
https://github.com/symfony-cmf/testing/issues/179

### CODE REVIEW

**Comment by dantleech** [comment]
Hmm looks like the Symfony CMF testing package needs SF4 support too.

Not sure about the memory issue on `5.x`, but happy to drop `5.x` from Travis.

Then it seems that we use a deprecated Doctrine ORM method - that would need to be looked at (but may well be in the CMF testing).

Also happy to drop the CMF testing dep if possible.

**Comment by chirimoya** [comment]
@dantleech symfony-cmf/testing 2.1.0-RC1 should work with symfony 4.0. will work on this PR after https://github.com/phpcr/phpcr-migrations/pull/8 is merged.

**Comment by chirimoya** [comment]
@dantleech can you have a look? tests are green now.

**Comment by dantleech** [comment]
Do you guys want to maintain?

**Comment by chirimoya** [comment]
@dantleech I guess we need to ;-) Can you add @wachterjohannes and @alexander-schranz as maintainer and maybe you find time to tag an new release. That would be awesome.

**Comment by dantleech** [comment]
Does this bundle still support 7.0? if so we should add that to travis

**Comment by dantleech** [comment]
Added @watcherjoannes and @alexander-schranz as collaborators (@chirimoya already has push access)

File: Resources/config/services.xml [hunk 1]
```diff
@@ -19,19 +19,19 @@
...
 
-        <service id="phpcr_migrations.command.status" class="DTL\Bundle\PhpcrMigrations\Command\StatusCommand">
+        <service id="phpcr_migrations.command.status" class="DTL\Bundle\PhpcrMigrations\Command\StatusCommand" public="true">
             <argument type="service" id="phpcr_migrations.version_storage" />
             <argument type="service" id="phpcr_migrations.version_finder" />
             <tag name="console.command" />
         </service>
 
-        <service id="phpcr_migrations.command.migrate" class="DTL\Bundle\PhpcrMigrations\Command\MigrateCommand">
+        <service id="phpcr_migrations.command.migrate" class="DTL\Bundle\PhpcrMigrations\Command\MigrateCommand" public="true">
```
Comment (dantleech) [inline Resources/config/services.xml]: does this break BC at all?

**Pull request merged**